packages/client/src/tools.js

```diff
@@ -869,20 +869,26 @@ export default class Tools {
   static async extractAbstract(url) {
     const metaExtractionUrl = 'https://meta-extraction-wivlbyt6ga-uc.a.run.app/metadata';
     const queryString = generateQueryString({ url });
-    const result = await fetch(`${metaExtractionUrl}?${queryString}`, {
-      method: 'GET',
-    });
-
-    if (result.status === 200) {
-      return result.json();
-    }
+    try {
+      const result = await fetch(`${metaExtractionUrl}?${queryString}`, {
+        method: 'GET',
+      });
 
-    const contentType = result.headers.get('content-type');
-    throw new OpenReviewError({
-      name: 'ExtractAbstractError',
-      message: (contentType && contentType.indexOf('application/json') !== -1) ? JSON.stringify(await result.json()) : await result.text(),
-      status: result.status || 500
-    });
+      if (result.status === 200) {
+        return result.json();
+      }
 
+      const contentType = result.headers.get('content-type');
+      throw new OpenReviewError({
+        name: 'ExtractAbstractError',
+        message: (contentType && contentType.indexOf('application/json') !== -1) ? JSON.stringify(await result.json()) : await result.text(),
+        status: result.status || 500
+      });
```

Comment:
I think this should go outside the `try catch` block. Otherwise, if an error is thrown here, we will get `message: [object object]`